lib/src/multires_stochastic_texture_synthesis.rs

```diff
@@ -761,6 +702,35 @@ impl Generator {
         let l2_precomputed = PrerenderedU8Function::new(metric_l2);
         let mut total_processed_pixels = 0;
         let max_workers = params.max_thread_count;
+        let mut has_fanned_out = false;
```

Comment:
Can you add a comment to clarify the "fanned out"? Perhaps something along the lines of "use a single R*-tree initially, and fan out to a grid thereof later".

Reply:
Will do!